### ISSUE TITLE
Check new model output and compare to old models

### ISSUE THREAD

**Issue by cchambe12** [lizzieinvancouver/ospree#348]
Step 1: rerun same model as we used in original budburst manuscript
Step 2: rerun model using only data before 2014
Step 3: rerun model using old species and complex requirements
Step 4: rerun model with combo of 2 and 3

**Comment by cchambe12** [comment]
Step 1:
`> m2lni.sum[grep("mu_", rownames(m2lni.sum)),]
                   mean    se_mean        sd      2.5%       25%       50%       75%      97.5%    n_eff      Rhat
mu_a_sp       29.875359 0.01655550 1.2961297 27.324123 29.017394 29.890311 30.701511 32.4644358 6129.320 0.9994977
mu_b_force_sp -4.914900 0.01417855 0.9805337 -6.795286 -5.559872 -4.923439 -4.259585 -2.9412009 4782.574 0.9998747
mu_b_photo_sp -1.573406 0.01215660 0.7057196 -2.929738 -2.047385 -1.574705 -1.094708 -0.1896565 3370.082 0.9995049
mu_b_chill_sp -4.513288 0.01619413 1.0885601 -6.600933 -5.245019 -4.525262 -3.793028 -2.3267995 4518.451 0.9998937`

**Comment by cchambe12** [comment]
Step 2:

 mean    se_mean          
mu_a_sp       30.086507 0.01872090 
mu_b_force_sp -5.400936 0.01338801 
mu_b_photo_sp -1.961453 0.01178286 
mu_b_chill_sp -5.338641 0.01691064

**Comment by cchambe12** [comment]
Step 3:

mu_a_sp       31.903170 0.02376869 
mu_b_force_sp -4.810116 0.02188890 
mu_b_photo_sp -1.698200 0.01612668 
mu_b_chill_sp -5.094670 0.02131322

**Comment by cchambe12** [comment]
Step 4:

mu_a_sp       31.928427 0.03072536 
mu_b_force_sp -4.856911 0.02414347 
mu_b_photo_sp -1.699426 0.01949806 
mu_b_chill_sp -5.106080 0.02324447

**Comment by cchambe12** [comment]
I added a step 5 that was more specific... I used old species code and removed all new studies.

mu_a_sp       30.972195 0.02807039 
mu_b_force_sp -4.683755 0.01921579 
mu_b_photo_sp -3.315974 0.02259264 
mu_b_chill_sp -7.547119 0.02549184

**Comment by cchambe12** [comment]
@AileneKane @lizzieinvancouver The suspense!!! But it seems that a new study is driving the strength of the forcing cue up. I _think_ it's the ramos17 study because when I just looked at data from 2014 or earlier in step 3 and 4, the chilling and forcing estimates were very similar. This dataset has 326 rows of data and is from 1995-1997.

**Comment by lizzieinvancouver** [comment]
@cchambe12 This is awesome work! (And exciting, watching it come in.) Thank you! Is ramos17 the olive study? Also (you might be doing this already), can you run step 2 or 3 but exclude ramos17?

**Comment by cchambe12** [comment]
@AileneKane @lizzieinvancouver Just kidding, we have a few new studies that are before 2014. I excluded ramos17 and it's not the bad guy. I'll keep checking and keep you posted!

**Comment by cchambe12** [comment]
@lizzieinvancouver @AileneKane Doesn't seem to be ramos17, man17, anzanello16, anzanello18 or nanninga17... but

Excluding malyshev18:
mu_a_sp       30.747696 0.02010689 
mu_b_force_sp -4.089160 0.01597065 
mu_b_photo_sp -2.233037 0.01616122 
mu_b_chill_sp -5.802668 0.01655264

**Comment by lizzieinvancouver** [comment]
@cchambe12 Thanks for your work on this! This seems like good news in that the climate data appears to be generating similar answers to your last climate data, do you agree (check me here, I am teaching so a little wonky-headed)?

I am looking malyshev18 and can see how it would have an influence: lots of data and 8 species (common ones too). But I am concerned the data are entered wrong. I can't dig deeper now, but Figure 2 (second and fourth panel) shows variation in 'chilling unit sums' but when I look at the data we have entered there's no variation in chilling or field sample date (that I see) ... @dbuona, were you the check on this data (I know Darwin entered it) or someone else? I think we do need to do some more new-data checks, I will start an issue.

@AileneKane I think we probably should move forward with your revision ... we really need to dig into our new results before we trust them the way we trust the results in the bb ms. Let me know what you think.

**Comment by cchambe12** [comment]
@lizzieinvancouver That was my takeaway! I agree we should move forward with main bb ms and we can trust the new climate data now. 

I think I was in charge of checking malyshev18 from Darwin so I can look at this too. But I agree it may be a good idea to check all of the new papers. By just excluding malyshev18, the chilling effect went up a lot but it still wasn't as high as when I excluded all new papers.

**Comment by lizzieinvancouver** [comment]
@cchambe12 Once we have finished cleaning malyshev, would you be willing to run the basic models again? Maybe with and without crops?

**Comment by cchambe12** [comment]
@lizzieinvancouver Much better! 

Without crops:
                   mean           sd
forcing       -5.21           0.86
photo         -2.45          0.77
chill            -9.10           2.02

All species:
                   mean           sd
forcing       -6.15           0.88
photo         -1.42           0.50
chill            -9.53           1.14

**Comment by lizzieinvancouver** [comment]
Yes! I would have expected our estimates would be more stable, as these suggest. Compared to the budburst ms:

Without crops:
mean
forcing -4.35
photo -2.95
chill  -8.35

All species:
forcing -6.17
photo 1.02
chill  -8.00

So overall the forcing and chilling effects both go up a little and the photo down (a hair) given the additional data, and we again see that the model with crops has a stronger forcing effect and weaker photoperiod effect. Cool! Thank you @cchambe12 and just letting @AileneKane know.

**Comment by AileneKane** [comment]
This is interesting and also good news that it is not as wildlly different as it was before. Thanks @cchambe12 @lizzieinvancouver  and @dbuona for your work on this!
